requestor/worldstate.go

```diff
@@ -141,3 +156,7 @@ func DailyDeals(platform string) {
 		fmt.Println("The information could not be fetched")
 	}
 }
+
+func timeTracking(startTime time.Time) {
+	fmt.Println("\n" + colorformat.ForeGrnRglr("Request Time") + " ... " + time.Since(startTime).Round(time.Millisecond).String() + "\n")
```

Comment:
Any reason as to why the time is fetched in milliseconds? :sweat_smile:

Reply:
it's just for formatting time in milliseconds, there is an example of how it works -> https://play.golang.org/p/tscRrEizKY8

Reply:
All right. That makes sense.